### PR TITLE
Add initial team page.

### PR DESCRIPTION
**Pull request by jwage**

### CODE REVIEW

**Comment by jwage** [comment]
@doctrine/doctrinecore Add yourself to this page if you want to be included. Add yourself to `app/config/sculpin_kernel.yml` and add your image to `source/images/team/{slug}.jpg`

File: source/team.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+title: Doctrine Team
+menuSlug: team
+---
+
+<h1>Doctrine Team</h1>
+
+<ul class="list-group">
+    {% for member in get_team() %}
+        <li class="list-group-item">
+            <img src="{{ get_asset_url('/images/team/' ~ member.slug ~ '.jpg', site.url) }}" class="rounded float-left mr-5" style="width: 50px;" alt="{{ member.name }}" />
```
[the remark this answers is not in the document]
Comment (jwage) [reply source/team.html]: What is that service called? Gravatar right?

File: app/src/Doctrine/Website/Twig/MainExtension.php [hunk 2]
```diff
@@ -10,10 +10,19 @@
 
 class MainExtension extends Twig_Extension
 {
+    /** @var array*/
```
[the remark this answers is not in the document]
Comment (Ocramius) [reply app/src/Doctrine/Website/Twig/MainExtension.php]: We would add `doctrine/coding-standard`. We can do it in a separate PR 👍

File: source/team.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (Ocramius) [reply source/team.html]: Yup, I totally forgot its name even if I also heavily rely on it: seems perfect for this scenario and is almost no maintenance

[the remark this answers is not in the document]
Comment (jwage) [reply source/team.html]: Updated to use gravatar 👍

**Comment by Ocramius** [comment]
@jwage I realise that you are going quite fast with the prototype, but please rebase rather than merging upstream 👌

**Comment by jwage** [comment]
@Ocramius rebased

**Comment by Ocramius** [comment]
:shipit:

**Pull request merged**